lib/terminal/hidden_windows.go

```diff
@@ -0,0 +1,19 @@
+// +build windows
+
+package terminal
+
+import (
+	"syscall"
+)
+
+// HideConsole hides the console window and activates another window
```

Comment:
Which windows gets activated? A random other window?

Reply:
Who (*maybe* Microsoft) knows.. Its taken from documentation at https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-showwindow. When I test it no window is active after rclone's consoles has been hidden.